numerics/next_test.cpp

```diff
@@ -0,0 +1,80 @@
+﻿#include "numerics/next.hpp"
+
+#include <limits>
+
+#include "geometry/sign.hpp"
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+
+namespace principia {
+namespace numerics {
+
+using geometry::Sign;
+using ::testing::Eq;
+
+class NextTest : public ::testing::Test {};
+
+TEST_F(NextTest, ExtendedRealLine) {
+  static_assert(NextUp(-std::numeric_limits<double>::infinity()) ==
+                -0x1.FFFFFFFFFFFFFp+1023);
+  static_assert(NextUp(-0x1.FFFFFFFFFFFFFp+1023) == -0x1.FFFFFFFFFFFFEp+1023);
+  // [...]
```

Comment:
No idea what this comment is trying to tell me.

Reply:
A few numbers were omitted in the progression from -∞ to +∞.